client/package.json

```diff
@@ -7,11 +7,14 @@
   "scripts": {
     "backend-dev": "python3.6 -m venv cellxgene && source cellxgene/bin/activate && yes | pip uninstall cellxgene || true && pip install -e .. && cellxgene launch ",
     "build": "npm run clean && webpack --config configuration/webpack/webpack.config.prod.js",
+    "e2e": "jest e2e",
     "dev": "npm run clean && webpack --config configuration/webpack/webpack.config.dev.js",
     "clean": "rimraf build",
     "start": "node server/development.js",
+    "start-server-for-test": "cellxgene launch -p 5000 ../example-dataset/pbmc3k.h5ad",
+    "ci": "start-server-and-test start-server-for-test :5000 e2e",
     "lint": "eslint src",
-    "test": "jest"
+    "test": "jest --testPathIgnorePatterns e2e"
```

Comment:
maybe rename this 'unittest' or something like that?

Reply:
good suggestion

Reply:
updated